install.sh

```diff
@@ -56,24 +56,13 @@ set -o pipefail  # Fail if one of the piped commands fails
 #   - KUBECONFIG_CONTEXT
 #     the context selected to interact with the cluster (defaults to the current one).
 #
-#   - LIQO_AGENT
-#     when this variable is 'true', it triggers the LiqoAgent (un)installer according to main options of this script.
-#     LiqoAgent is a desktop application (currently just for linux-based desktop environments) providing an easy way
-#     to manage Liqo through a simple graphical user interface. Learn more at https://github.com/liqotech/liqo-agent.
-#   - LIQO_DASHBOARD
```

Comment:
Should also remove the DASHBOARD_HOSTNAME variable from the description above and from the help message